app/main.py

```diff
@@ -1,5 +1,108 @@
+from __future__ import annotations
+
+
 class Cargo:
-    def __init__(self, weight):
+    def __init__(self, weight: float) -> None:
+        self.weight = weight
+
+
+class BaseRobot:
+    def __init__(
+            self,
+            name: str,
+            weight: float,
+            coords: list = None
+    ) -> None:
+        self.name = name
         self.weight = weight
+        self.coords = []
+        if coords is None:
+            self.coords.append(0)
```

Comment:
Ти можеш робити присвоєння списків тут 
```suggestion
            self.coords = [0, 0]
```